GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3330,6 +3330,71 @@ internal TestAccessor(FormCommit formCommit)
                 string commitMessageText, bool usingCommitTemplate, bool ensureCommitMessageSecondLineEmpty)
                     => FormCommit.FormatCommitMessageFromTextBox(commitMessageText, usingCommitTemplate, ensureCommitMessageSecondLineEmpty);
         }
+
+        private void toWindowsCRLFToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            ConvertEndOfLines(Environment.NewLine);
+        }
+
+        private void ConvertEndOfLines(string eol)
```

Comment:
This non-trivial function should be tested. (Another candidate for `ApprovalTests.net`?)